### PR TITLE
chore: Add safety function with default value

### PR DESCRIPTION
**Pull request by PatrykGala**
## Before submitting checklist

- [ ] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

**Comment by PatrykGala** [comment]
E2e:  https://github.com/neptune-ai/neptune-client/actions/runs/6480870026

File: src/neptune/handler.py [hunk 1]
```diff
@@ -164,6 +165,7 @@ def get_root_object(self) -> "NeptuneObject":
         """
         return self._container
 
+    @safe_function()
```
Comment (aniezurawski) [inline src/neptune/handler.py:168]: We should decorate `__getitem__` and `__setitem__` as well, shouldn't we?

File: src/neptune/metadata_containers/metadata_container.py [hunk 2]
```diff
@@ -437,6 +446,7 @@ def stop(self, *, seconds: Optional[Union[float, int]] = None) -> None:
             self._state = ContainerState.STOPPED
             self._forking_cond.notify_all()
 
+    @safe_function("started")
```
Comment (aniezurawski) [inline src/neptune/metadata_containers/metadata_container.py:449]: I would rather return `None` here.

File: src/neptune/metadata_containers/run.py [hunk 3]
```diff
@@ -101,7 +100,6 @@
 from neptune.types.mode import Mode
 
 
-@safety_decorator
 class Run(MetadataContainer):
```
Comment (aniezurawski) [inline src/neptune/metadata_containers/run.py:103]: There are some public methods in `Run`, `Model`, etc.

File: src/neptune/metadata_containers/safe_container.py [hunk 4]
```diff
@@ -13,41 +13,41 @@
...
-def safety_decorator(cls: Any) -> Any:
-    if _SAFETY_MODE:
-        for name, method in inspect.getmembers(cls):
-            if (not inspect.ismethod(method) and not inspect.isfunction(method)) or inspect.isbuiltin(method):
-                continue
-            setattr(cls, name, _safe_function(method))
-        return cls
-    else:
-        return cls
+_SAFETY_MODE = os.getenv(NEPTUNE_SAFETY_MODE, "True").lower() in ("true", "1", "t")
```
[the remark this answers is not in the document]
Comment (aniezurawski) [reply src/neptune/metadata_containers/safe_container.py:30]: Yup.

File: src/neptune/metadata_containers/safe_container.py [hunk 5]
```diff
@@ -13,41 +13,41 @@
...
+
+
+def safe_function(default_return_value: Any = None) -> Any:
```
Comment (aniezurawski) [inline src/neptune/metadata_containers/safe_container.py:33]: Look good but I would like @Raalsky to take a look.

**Review by aniezurawski — CHANGES_REQUESTED** [review]

**Review by aniezurawski — APPROVED** [review]

**Pull request merged**